dev-qt/qtgui/qtgui-5.8.9999.ebuild

```diff
@@ -137,12 +138,14 @@ src_prepare() {
 	use dbus || sed -i -e 's/contains(QT_CONFIG, dbus)/false/' \
 		src/platformsupport/platformsupport.pro || die
 
+	qt_use_disable_config tuio udpsocket src/plugins/generic/generic.pro
+
 	qt_use_disable_mod ibus dbus \
```

Comment:
I moved it because we traditionally we don't seem to have comments on function calls like this. I have no preference so I'm happy to duplicate the comment or group them together or whatever - just let me know what you prefer.